x-pack/legacy/plugins/index_lifecycle_management/public/np_ready/application/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { Provider } from 'react-redux';
+import { DocLinksStart, ToastsSetup } from 'src/core/public';
+
+import { App } from './app';
+import { indexLifecycleManagementStore } from './store';
+import { init as initHttp } from './services/http';
+import { init as initNavigation } from './services/navigation';
+import { init as initDocumentation } from './services/documentation';
+import { init as initUiMetric } from './services/ui_metric';
+import { init as initNotification } from './services/notification';
+
+export interface LegacySetup {
+  http: any;
+  redirect: any;
+  fatalError: any;
+  createUiStatsReporter: any;
+}
+
+interface AppDependencies {
+  legacy: LegacySetup;
+  I18nContext: any;
+  toasts: ToastsSetup;
+  prependBasePath: any;
+  docLinks: DocLinksStart;
+  element: HTMLElement;
+}
+
+export const boot = (appDependencies: AppDependencies) => {
```

Comment:
[By convention](https://github.com/elastic/kibana/blob/master/src/core/CONVENTIONS.md#applications) this should be named `renderApp`.